{"uuid": "0f0b1d2f-5776-47bf-a096-a723ad087c42", "children": ["95fa231d-0527-44ed-861c-b4bece73e249"], "befores": [{"name": "order", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <button type=\"button\" class=\"Modal_modal__close_modified__3V5XS Modal_modal__close__TnseK\">...</button> is not clickable at point (1202, 192). Other element would receive the click: <div class=\"Modal_modal_overlay__x2ZCr\"></div>\n  (Session info: chrome=126.0.6478.183)\nStacktrace:\n0   chromedriver                        0x0000000105306078 chromedriver + 5169272\n1   chromedriver                        0x00000001052fdf4a chromedriver + 5136202\n2   chromedriver                        0x0000000104e7a36c chromedriver + 402284\n3   chromedriver                        0x0000000104ece6b6 chromedriver + 747190\n4   chromedriver                        0x0000000104ecc4e8 chromedriver + 738536\n5   chromedriver                        0x0000000104ec9ef7 chromedriver + 728823\n6   chromedriver                        0x0000000104ec8d12 chromedriver + 724242\n7   chromedriver                        0x0000000104ebc562 chromedriver + 673122\n8   chromedriver                        0x0000000104eeaab2 chromedriver + 862898\n9   chromedriver                        0x0000000104ebbed8 chromedriver + 671448\n10  chromedriver                        0x0000000104eeac6e chromedriver + 863342\n11  chromedriver                        0x0000000104f09f57 chromedriver + 991063\n12  chromedriver                        0x0000000104eea853 chromedriver + 862291\n13  chromedriver                        0x0000000104eba5c6 chromedriver + 665030\n14  chromedriver                        0x0000000104ebae4e chromedriver + 667214\n15  chromedriver                        0x00000001052c8c90 chromedriver + 4918416\n16  chromedriver                        0x00000001052cdc8d chromedriver + 4938893\n17  chromedriver                        0x00000001052ce365 chromedriver + 4940645\n18  chromedriver                        0x00000001052a9d74 chromedriver + 4791668\n19  chromedriver                        0x00000001052ce659 chromedriver + 4941401\n20  chromedriver                        0x000000010529b544 chromedriver + 4732228\n21  chromedriver                        0x00000001052ee828 chromedriver + 5072936\n22  chromedriver                        0x00000001052ee9e7 chromedriver + 5073383\n23  chromedriver                        0x00000001052fdafe chromedriver + 5135102\n24  libsystem_pthread.dylib             0x00007ff81337118b _pthread_start + 99\n25  libsystem_pthread.dylib             0x00007ff81336cae3 thread_start + 15\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1122, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/_pytest/fixtures.py\", line 884, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/dunaeva.ev/Desktop/Diplom/Diplom_3/conftest.py\", line 111, in order\n    create_order(index_page)\n  File \"/Users/dunaeva.ev/Desktop/Diplom/Diplom_3/helpers.py\", line 5, in create_order\n    index_page.click_cross_button_in_popup_window()\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/dunaeva.ev/Desktop/Diplom/Diplom_3/pages/index_page.py\", line 24, in click_cross_button_in_popup_window\n    self.click_by_element(IndexPageLocators.BUTTON_CROSS_IN_POPUP_WINDOW)\n  File \"/Users/dunaeva.ev/Desktop/Diplom/Diplom_3/pages/base_page.py\", line 27, in click_by_element\n    WDWait(self.web_drv, timeout).until(ec.element_to_be_clickable(locator)).click()\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Добавляем ингредиент в заказ", "status": "passed", "parameters": [{"name": "index", "value": "1"}], "start": 1721429022802, "stop": 1721429023461}, {"name": "Добавляем ингредиент в заказ", "status": "passed", "parameters": [{"name": "index", "value": "4"}], "start": 1721429023461, "stop": 1721429023568}, {"name": "Клик на кнопку 'Оформить заказ'", "status": "passed", "start": 1721429023568, "stop": 1721429023628}, {"name": "Клик на крестик во всплывающем окне", "status": "passed", "start": 1721429023629, "stop": 1721429025242}, {"name": "Добавляем ингредиент в заказ", "status": "passed", "parameters": [{"name": "index", "value": "1"}], "start": 1721429025243, "stop": 1721429025360}, {"name": "Добавляем ингредиент в заказ", "status": "passed", "parameters": [{"name": "index", "value": "4"}], "start": 1721429025360, "stop": 1721429025465}, {"name": "Клик на кнопку 'Оформить заказ'", "status": "passed", "start": 1721429025466, "stop": 1721429025522}, {"name": "Клик на крестик во всплывающем окне", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <button type=\"button\" class=\"Modal_modal__close_modified__3V5XS Modal_modal__close__TnseK\">...</button> is not clickable at point (1202, 192). Other element would receive the click: <div class=\"Modal_modal_overlay__x2ZCr\"></div>\n  (Session info: chrome=126.0.6478.183)\nStacktrace:\n0   chromedriver                        0x0000000105306078 chromedriver + 5169272\n1   chromedriver                        0x00000001052fdf4a chromedriver + 5136202\n2   chromedriver                        0x0000000104e7a36c chromedriver + 402284\n3   chromedriver                        0x0000000104ece6b6 chromedriver + 747190\n4   chromedriver                        0x0000000104ecc4e8 chromedriver + 738536\n5   chromedriver                        0x0000000104ec9ef7 chromedriver + 728823\n6   chromedriver                        0x0000000104ec8d12 chromedriver + 724242\n7   chromedriver                        0x0000000104ebc562 chromedriver + 673122\n8   chromedriver                        0x0000000104eeaab2 chromedriver + 862898\n9   chromedriver                        0x0000000104ebbed8 chromedriver + 671448\n10  chromedriver                        0x0000000104eeac6e chromedriver + 863342\n11  chromedriver                        0x0000000104f09f57 chromedriver + 991063\n12  chromedriver                        0x0000000104eea853 chromedriver + 862291\n13  chromedriver                        0x0000000104eba5c6 chromedriver + 665030\n14  chromedriver                        0x0000000104ebae4e chromedriver + 667214\n15  chromedriver                        0x00000001052c8c90 chromedriver + 4918416\n16  chromedriver                        0x00000001052cdc8d chromedriver + 4938893\n17  chromedriver                        0x00000001052ce365 chromedriver + 4940645\n18  chromedriver                        0x00000001052a9d74 chromedriver + 4791668\n19  chromedriver                        0x00000001052ce659 chromedriver + 4941401\n20  chromedriver                        0x000000010529b544 chromedriver + 4732228\n21  chromedriver                        0x00000001052ee828 chromedriver + 5072936\n22  chromedriver                        0x00000001052ee9e7 chromedriver + 5073383\n23  chromedriver                        0x00000001052fdafe chromedriver + 5135102\n24  libsystem_pthread.dylib             0x00007ff81337118b _pthread_start + 99\n25  libsystem_pthread.dylib             0x00007ff81336cae3 thread_start + 15\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/dunaeva.ev/Desktop/Diplom/Diplom_3/pages/index_page.py\", line 24, in click_cross_button_in_popup_window\n    self.click_by_element(IndexPageLocators.BUTTON_CROSS_IN_POPUP_WINDOW)\n  File \"/Users/dunaeva.ev/Desktop/Diplom/Diplom_3/pages/base_page.py\", line 27, in click_by_element\n    WDWait(self.web_drv, timeout).until(ec.element_to_be_clickable(locator)).click()\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1721429025522, "stop": 1721429026612}], "start": 1721429022802, "stop": 1721429026619}], "start": 1721429022802, "stop": 1721429026746}